cmd/system-probe/common/common_nix.go

```diff
@@ -7,7 +7,28 @@
 
 package common
 
+import (
+	ebpfkernel "github.com/DataDog/datadog-agent/pkg/security/ebpf/kernel"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
 const (
 	//nolint:revive // TODO(EBPF) Fix revive linter
 	DefaultLogFile = "/var/log/datadog/system-probe.log"
 )
+
+// DisablePESUnsupportedKernel returns true if network_process needs to be disabled due to unsupported kernel version
+func DisablePESUnsupportedKernel(isEnabled bool) bool {
+	if !isEnabled {
+		return false
+	}
+	kernelVersion, err := ebpfkernel.NewKernelVersion()
+	if err != nil {
```

Comment:
nit: you can get rid of the `else if` if you return early here with `return false`.